app/views/communicart_mailer/cart_notification_email.html.erb

```diff
@@ -31,7 +31,9 @@
     <tr id="approval-actions">
       <td colspan=2 align="center">
         <p>
-          <%= link_to "Approve all items with one click", approve_all_url(@approval), class: 'form-button' %>
+          <%= link_to "Approve all items with one click", approve_all_url(@approval, true), class: 'form-button' %>
+          <%= link_to "Reject all items with one click", approve_all_url(@approval, false), class: 'form-button' %>
```

Comment:
Could the buttons just say "approve" and "reject"? Pretty sure we don't have item-level rejection at this point anyway.